test/stub-test.js

```diff
@@ -3781,6 +3781,29 @@ describe("stub", function () {
             assert.equals(myFunc.prop, "rawString");
         });
 
+        it("allows stubbing getters", function () {
+            const y = {
+                get foo() {
+                    return "bar";
+                },
+            };
+            refute.exception(function () {
+                createStub(y, "foo").value("bar");
+            });
+        });
```

Comment:
The original issue (#2629) does not deal with this issue, so this seems more like we are documenting existing (non-intentional) behavior? Dan does talk about getters, but that is the conventional Java-bean like getters, which are methods.